src/App.js

```diff
@@ -1,23 +1,13 @@
-import logo from './logo.svg';
+import Card from './DiaryCard/DiaryCard'
```

Comment:
use same name for every place that each component use because this can be difficult to identity.